assets/scripts/utils/transform.js

```diff
@@ -1,21 +1,51 @@
-export function transform(el, transformValue){
-    el.style.webkitTransform = transformValue;
-    el.style.msTransform = transformValue;
-    el.style.transform = transformValue;
+/**
+ * Transform function
+ * @param {HTMLElement} $el         - DOM element
+ * @param {number}      value       - transform value
+ * @return {void}
+ */
+
+const transform = ($el, value) => {
+    $el.style.msTransform = value
+    $el.style.webkitTransform = value
+    $el.style.MozTransform = value
+    $el.style.OTransform = value
+    $el.style.transform = value
 }
```

Comment:
I think we can drop this one? Transform doesn't need prefixes for the modern browsers we agreed to support